core/keyserver/epochs_test.go

```diff
@@ -38,47 +36,32 @@ const (
 	domainID = "domain"
 )
 
-func updates(t *testing.T, start, end int) []*pb.EntryUpdate {
-	if start > end {
-		t.Fatalf("start=%v > end=%v", start, end)
-	}
-	kvs := make([]*pb.EntryUpdate, 0, end-start)
+func genMutations(start, end int) []*pb.Entry {
+	mutations := make([]*pb.Entry, 0, end-start)
 	for i := start; i <= end; i++ {
-		kvs = append(kvs, &pb.EntryUpdate{
-			Mutation: &pb.Entry{
-				Index:      []byte(fmt.Sprintf("key_%v", i)),
-				Commitment: []byte(fmt.Sprintf("value_%v", i)),
-			},
+		mutations = append(mutations, &pb.Entry{
+			Index:      []byte(fmt.Sprintf("key_%v", i)),
+			Commitment: []byte(fmt.Sprintf("value_%v", i)),
 		})
 	}
-	return kvs
-}
-
-func prepare(ctx context.Context, t *testing.T, domainID string, mutations mutator.MutationStorage, tmap *fake.MapServer) {
-	createEpoch(ctx, t, domainID, mutations, tmap, 1, 1, 6)
-	createEpoch(ctx, t, domainID, mutations, tmap, 2, 7, 10)
+	return mutations
 }
 
-func createEpoch(ctx context.Context, t *testing.T, domainID string, mutations mutator.MutationStorage, tmap *fake.MapServer, epoch int64, start, end int) {
-	kvs := updates(t, start, end)
-	for _, kv := range kvs {
-		if _, err := mutations.Write(ctx, domainID, kv); err != nil {
-			t.Fatalf("mutations.Write failed: %v", err)
+func prepare(ctx context.Context, domainID string, m mutator.MutationStorage, tmap *fake.MapServer) error {
+	for _, rev := range []struct {
+		epoch      int64
+		start, end int
+	}{
+		{epoch: 1, start: 1, end: 6},
+		{epoch: 2, start: 7, end: 10},
```

Comment:
This table of epoch/start/end values look like test data but aren't part of the tests that follow.
  Should the table be passed in?

Reply:
I've moved this to the test itself.